### PR TITLE
tiup: update docs of tiup-bench

### PR DESCRIPTION
**Pull request by zyguan**
Signed-off-by: zyguan <zhongyangguan@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Keeping tiup-bench docs up-to-date. see https://github.com/pingcap/docs-cn/pull/11838 .

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11838
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-bench.md [hunk 1]
```diff
@@ -62,14 +63,18 @@ Available Commands:
   run         Run workload
 
 Flags:
-      --check-all        Run all consistency checks
-  -h, --help             help for tpcc
-      --parts int        Number to partition warehouses (default 1)
-      --warehouses int   Number of warehouses (default 10)
+      --check-all            Run all consistency checks
+  -h, --help                 help for tpcc
```
Comment (qiancai) [inline tiup/tiup-bench.md:67]:
```suggestion
  -h, --help                 Help for TPC-C
```

File: tiup/tiup-bench.md [hunk 2]
```diff
@@ -62,14 +63,18 @@ Available Commands:
...
+      --partition-type int   Partition type: 1 - HASH, 2 - RANGE, 3 - LIST (HASH-like), 4 - LIST (RANGE-like) (default 1)
```
Comment (qiancai) [inline tiup/tiup-bench.md:68]:
```suggestion
      --partition-type int   Partition type: 1 - HASH, 2 - RANGE, 3 - LIST (HASH-like), 4 - LIST (RANGE-like) (default to 1)
```

File: tiup/tiup-bench.md [hunk 3]
```diff
@@ -62,14 +63,18 @@ Available Commands:
...
+      --parts int            Number to partition warehouses (default 1)
```
Comment (qiancai) [inline tiup/tiup-bench.md:69]: 请教下，这个参数是代表“各个分区里仓库的数量”吗，即 Number of warehouses in each partition？

File: tiup/tiup-bench.md [hunk 4]
```diff
@@ -62,14 +63,18 @@ Available Commands:
...
+      --warehouses int       Number of warehouses (default 10)
```
Comment (qiancai) [inline tiup/tiup-bench.md:70]:
```suggestion
      --warehouses int       Number of warehouses (default to 10)
```

File: tiup/tiup-bench.md [hunk 5]
```diff
@@ -62,14 +63,18 @@ Available Commands:
...
+
 ```
 
 ### Test procedures
 
+Here we show simplified steps for running a TPC-C test. Please refer to [How to Run TPC-C Test on TiDB](/benchmark/benchmark-tidb-using-tpcc.md) for a more complete guide.
```
Comment (qiancai) [inline tiup/tiup-bench.md:76]:
```suggestion
The following provides simplified steps for running a TPC-C test. For detailed steps, see [How to Run TPC-C Test on TiDB](/benchmark/benchmark-tidb-using-tpcc.md).
```

File: tiup/tiup-bench.md [hunk 6]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
+      --count int             Total execution count, 0 means infinite
+  -D, --db string             Database name (default "test")
+  -d, --driver string         Database driver: mysql, postgres (default "mysql")
+      --dropdata              Cleanup data before prepare
+  -H, --host strings          Database host (default [127.0.0.1])
+      --ignore-error          Ignore error when running workload
+      --interval duration     Output interval time (default 10s)
+      --isolation int         Isolation Level 0: Default, 1: ReadUncommitted,
+                              2: ReadCommitted, 3: WriteCommitted, 4: RepeatableRead,
+                              5: Snapshot, 6: Serializable, 7: Linerizable
```
Comment (qiancai) [inline tiup/tiup-bench.md:33]:
```suggestion
                              2: ReadCommitted; 3: WriteCommitted; 4: RepeatableRead; 
                              5: Snapshot; 6: Serializable; 7: Linerizable)
```

File: tiup/tiup-bench.md [hunk 7]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
+      --max-procs int         runtime.GOMAXPROCS of golang, the limits of how many cores can be used
+      --output string         Output style, valid values can be { plain | table | json } (default "plain")
```
Comment (qiancai) [inline tiup/tiup-bench.md:35]:
```suggestion
      --output string         Output style. Valid values can be { plain | table | json } (default "plain")
```

File: tiup/tiup-bench.md [hunk 8]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
-  -T, --threads int         Thread concurrency (default 16)
-      --time duration       Total execution time (default 2562047h47m16.854775807s)
-  -U, --user string         Database user (default "root")
 ```
+  -t, --acThreads int         OLAP client concurrency, only for CH-benCHmark (default 1)
+      --conn-params string    Session variables, e.g. --conn-params tidb_isolation_read_engines='tiflash' for TiDB, --conn-params sslmode=disable for PostgreSQL
+      --count int             Total execution count, 0 means infinite
+  -D, --db string             Database name (default "test")
+  -d, --driver string         Database driver: mysql, postgres (default "mysql")
+      --dropdata              Cleanup data before prepare
```
Comment (qiancai) [inline tiup/tiup-bench.md:27]:
```suggestion
      --dropdata              Clean up historical data before preparing 
```

File: tiup/tiup-bench.md (hunk 3 above)
Comment (zyguan) [reply tiup/tiup-bench.md:69]: 不是，是 Number of partitions 的意思...

**Review by dbsid — APPROVED** [review]
lgtm

File: tiup/tiup-bench.md [hunk 9]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
+tiup bench tpch   # Benchmark a database using TPC-H
+tiup bench ch     # Benchmark a database using CH-benCHmark
+tiup bench ycsb   # Benchmark a database using YCSB
+tiup bench rawsql # Benchmark a database using arbitrary SQL files
 ```
-Benchmark database with different workloads
 
-Usage:
-  tiup bench [command]
+`tpcc`, `tpch`, `ch`, `rawsql` share the following common command flags. However, `ycsb` is mainly configured by a `.properties` file, which is described in its [usage guide](https://github.com/pingcap/go-ycsb#usage).
```
Comment (hfxsd) [inline tiup/tiup-bench.md:19]:
```suggestion
`tpcc`, `tpch`, `ch`, and `rawsql` share the following common command flags. However, `ycsb` is mainly configured by a `.properties` file, which is described in its [usage guide](https://github.com/pingcap/go-ycsb#usage).
```

File: tiup/tiup-bench.md [hunk 10]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
-  -U, --user string         Database user (default "root")
 ```
+  -t, --acThreads int         OLAP client concurrency, only for CH-benCHmark (default to 1)
+      --conn-params string    Session variables, such as setting `--conn-params tidb_isolation_read_engines='tiflash'` for TiDB queries and setting `--conn-params sslmode=disable` for PostgreSQL connections
+      --count int             Total execution count (0 means infinite count)
+  -D, --db string             Database name (default to "test")
+  -d, --driver string         Database driver: mysql, postgres (default to "mysql")
+      --dropdata              Clean up historical data before preparing
+  -H, --host strings          Database host (default to [127.0.0.1])
+      --ignore-error          Ignore error when running workload
```
Comment (hfxsd) [inline tiup/tiup-bench.md:29]:
```suggestion
      --ignore-error          Ignore errors when running workload
```

File: tiup/tiup-bench.md [hunk 11]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
+      --interval duration     Output interval time (default to 10s)
+      --isolation int         Isolation Level (0: Default; 1: ReadUncommitted;
+                              2: ReadCommitted; 3: WriteCommitted; 4: RepeatableRead;
+                              5: Snapshot; 6: Serializable; 7: Linerizable)
+      --max-procs int         runtime.GOMAXPROCS of golang, the limits of how many cores can be used
+      --output string         Output style. Valid values can be { plain | table | json } (default to "plain")
+  -p, --password string       Database password
+  -P, --port ints             Database port (default to [4000])
+      --pprof string          Address of pprof endpoint
+      --silence               Don't print error when running workload
```
Comment (hfxsd) [inline tiup/tiup-bench.md:39]:
```suggestion
      --silence               Don't print errors when running workload
```

File: tiup/tiup-bench.md [hunk 12]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
+  -S, --statusPort int        Database status port (default to 10080)
+  -T, --threads int           Thread concurrency (default to 1)
+      --time duration         Total execution time (default to 2562047h47m16.854775807s)
+  -U, --user string           Database user (default to "root")
+```
+
+- You can pass comma-separated values to `--host` and `--port` to enable client-side load balancing. For example, when specifying `--host 172.16.4.1,172.16.4.2 --port 4000,4001`, the program will connect to 172.16.4.1:4000, 172.16.4.1:4001, 172.16.4.2:4000, 172.16.4.2:4001, chosen in round-robin fashion.
```
Comment (hfxsd) [inline tiup/tiup-bench.md:46]:
```suggestion
- You can pass comma-separated values to `--host` and `--port` to enable client-side load balancing. For example, when you specify `--host 172.16.4.1,172.16.4.2 --port 4000,4001`, the program will connect to 172.16.4.1:4000, 172.16.4.1:4001, 172.16.4.2:4000, and 172.16.4.2:4001, chosen in round-robin fashion.
```

File: tiup/tiup-bench.md [hunk 13]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
...
+- `--conn-params` must follow the format of [query string](https://en.wikipedia.org/wiki/Query_string). Different databases might have different parameters. For example:
+    - `--conn-params tidb_isolation_read_engines='tiflash'` forces TiDB to read from TiFlash.
+    - `--conn-params sslmode=disable` disables SSL when you connect to PostgreSQL.
+- When running CH-benCHmark, you can use `--ap-host`, `--ap-port`, `--ap-conn-params` to specify a standalone TiDB server for OLAP queries.
```
Comment (hfxsd) [inline tiup/tiup-bench.md:50]:
```suggestion
- When running CH-benCHmark, you can use `--ap-host`, `--ap-port`, and `--ap-conn-params` to specify a standalone TiDB server for OLAP queries.
```

File: tiup/tiup-bench.md [hunk 14]
```diff
@@ -83,34 +88,36 @@ Flags:
...
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    tiup bench tpcc --warehouses 4 check
+    tiup bench tpcc --warehouses 4 cleanup
     ```
 
-5. Generate the CSV file:
+Preparing data via SQL might be slow when you want to run a benchmark with a large data set. In that case, you can generate data in the CSV format by the following commands and then import it to TiDB via [TiDB lightning](/tidb-lightning/tidb-lightning-overview.md).
```
Comment (hfxsd) [inline tiup/tiup-bench.md:110]:
```suggestion
Preparing data via SQL might be slow when you want to run a benchmark with a large data set. In that case, you can generate data in the CSV format by the following commands and then import it to TiDB via [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md).
```

File: tiup/tiup-bench.md [hunk 15]
```diff
@@ -6,49 +6,50 @@ aliases: ['/docs/dev/tiup/tiup-bench/','/docs/dev/reference/tools/tiup/bench/']
```
Comment (hfxsd) [inline tiup/tiup-bench.md:6]: L3 的 summary 也改下：
Learn how to stress test TiDB with TPC-C, TPC-H, CH, RawSQL, and YCSB workloads using TiUP

**Review by qiancai — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**